LayoutTests/accessibility/full-size-kana-untransformed.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../resources/accessibility-helper.js"></script>
+<script src="../resources/js-test.js"></script>
+<meta charset=utf-8>
+</head>
+<body>
+
+<div id="text">
+<strong style="text-transform: full-size-kana" id="full-size-kana-text">じゅう</strong> must read the same as じゅう not じゆう.
+</div>
+
+<script>
+var output = "This tests that text with full-size-kana returns the untransformed text to Accessibility APIs.\n\n";
+
+if (window.accessibilityController) {
+    var textElement = accessibilityController.accessibleElementById("text");
+    var textMarkerRange = textElement.textMarkerRangeForElement(textElement);
+
+    var startMarker = textElement.startTextMarkerForTextMarkerRange(textMarkerRange);
+    var endMarker = textElement.endTextMarkerForTextMarkerRange(textMarkerRange);
+    textMarkerRange = textElement.textMarkerRangeForMarkers(startMarker, endMarker);
```

Comment:
Why are we re-creating the range here?

Reply:
I was following a pattern from other tests, but I think we don't have to reconstruct. I'll remove that.